Modules/Promotion/Http/Services/PromotionService.php

```diff
@@ -192,4 +245,17 @@ public function delete(string $promotionKey): array
 
     }
 
+    /**
+     * Get a listing of the promotion featured ads
+     *
+     * @return array
+     */
+    public function getFeatures(): array
+    {
+        $features = Feature::get();
```

Comment:
@arijit1111 Why we are getting the all `Feature`? How `Feature` are actually created?

Reply:
Feature are Featured ad for promotion, table created by migration but data inserted manually

Reply:
@arijit1111 So the questions is, why we are getting all of them instead for only the active user?